packages/components/src/form-toggle/style.scss

```diff
@@ -4,8 +4,9 @@ $toggle-border-width: 2px;
 
 .components-form-toggle {
 	position: relative;
+	margin-right: $grid-size-large;
```

Comment:
It seems weird to add a margin right globally too all toggles? Why do we need this?

Reply:
I moved it to the `ToggleControl` style as it seems that this was specific to that component instead.

Reply:
The thing is, the label is clickable. Usually, the bigger the clickable area, the easier it is to hit and therefore the more user-friendly it is. Because this is a flex container, the setting fixed margin can stretch the element. This was why.

I can't test if the new rules achieve the same, and if they don't that's not the end of the world. Thanks for fixing. ♥️

Reply:
Don't worry, I applied the exact same style, I just moved it elsewhere to keep the genericity of the form toggle component.

Reply:
Not sure how this affects the label's clickable area though.

Reply:
It's fine, it's not visible, so it's an enhancement that can happen later.